### DISCUSSION TITLE
Webserver to run in background

### DISCUSSION THREAD

**Discussion by Predrog** [nicolargo/glances#1940] [Q&A] [answered]
It run on an Ubuntu server 20.04.3.

When i run Glances -w it start up perfectly and i can reach it from another computer. 
But if I want to get back to normal terminal command line it only react to ctrl + c. (I tried to just close the ssh session but same effect.) After that the browser just say disconnected.

How do I need to run the webserver to run in background?
Did I miss something?

**Comment by nicolargo** [comment, marked as answer]
You need to run the process in background:

https://www.tecmint.com/run-linux-command-process-in-background-detach-process/

Example:

```
glances -w &
```

**Reply by Predrog** [reply]
Thank you
